src/util.cpp

```diff
@@ -532,7 +532,11 @@ CAboutDlg::CAboutDlg ( QWidget* parent ) : CBaseDlg ( parent )
                               "<p><b>" +
                               tr ( "Slovak" ) +
                               "</b></p>"
-                              "<p>Jose Riha (<a href=\"https://github.com/jose1711\">jose1711</a>)</p>" );
+                              "<p>Jose Riha (<a href=\"https://github.com/jose1711\">jose1711</a>)</p>" +
+                              "<p><b>" +
+                              tr ( "Simplified Chinese" ) +
```

Comment:
This would fall `clang-format` check but using the result from `clang-format` will make the style not aligned to the format used above in the same file. What should I do or did I missed something?

Reply:
Hmm. Maybe the clang format versions are different.

Have a look at the patch here: https://github.com/jamulussoftware/jamulus/pull/1927/checks?check_run_id=3154583850